src/buildTree.js

```diff
@@ -12,13 +12,14 @@ const INHERITABLE_STYLES = [
   'fontSize',
   'fontStyle',
   'fontWeight',
+  'textAlign',
+  'textDecoration',
```

Comment:
we mirror the React Native APIs as much as possible — can you change this to `textDecorationLine` to match?